WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -79,13 +79,13 @@
                 <div class="row row-eq-height">
                     <div class="col-12 col-sm-5">
                         <div class="hero-image">
-                            <span class="text-lowercase mb-0 px-2 pb-5 mobile-heading d-sm-none">Unfairly Private</span>
+                            <span class="text-lowercase mb-0 px-2 pb-5 mobile-heading d-sm-none">Your Bitcoin privacy wallet</span>
```

Comment:
How about "Privacy, Transparency, Love" for a slogen? 😄

Reply:
It's just the slogan for the mobile view